### PR TITLE
Add details for egress IP address usage

### PR DESCRIPTION
**Pull request by jboxman**
This PR clarifies how egress IP address assignment works.

- https://bugzilla.redhat.com/show_bug.cgi?id=1794187

### CODE REVIEW

**Comment by jboxman** [comment]
@danwinship, so I updated the description of each egress IP assignment approach to the following ([Egress IP address assignment for project egress traffic](https://bz-1794187--ocpdocs.netlify.com/openshift-enterprise/latest/networking/openshift_sdn/assigning-egress-ips.html#nw-egress-ips-about_egress-ips)):

> You can assign egress IP addresses to namespaces by setting the `egressIPs` parameter of the `NetNamespace` resource. After an egress IP is associated with a project, OpenShift SDN allows you to assign egress IPs to hosts in two ways:

> * In the _automatically assigned_ approach, an egress IP address range is assigned to a node.
You set the `egressCIDRs` parameter of each node's `HostSubnet` resource to indicate the range of egress IP addresses that can be hosted by a node.
Only a single egress IP address is supported when using the automatic assignment mode.
This approach works best for clusters installed in environments with flexibility in associating additional IP addresses with nodes.

> * In the _manually assigned_ approach, a list of one or more egress IP address is assigned to a node.
You set the `egressIPs` parameter of each node's `HostSubnet` resource to indicate the IP addresses that can be hosted by a node.
Multiple egress IP addresses are supported when using the manual assignment mode.
This approach is recommended for clusters installed in public cloud environments, where there can be limitations on associating additional IP addresses with nodes.

**Comment by danwinship** [comment]
>     * Only a single egress IP address is supported when using the automatic assignment mode.

Only a single egress IP address *per namespace*. Likewise with the multiple IPs in the manual mode.

Also you should probably explain the contrasting HA modes here too. So for automatically-assigned, something like "if the node hosting the namespace's egress IP goes down, OCP will reassign the egress IP to another node with a compatible `egressCIDRs`". And for manually-assigned, "when a namespace has multiple egress IPs, if the node hosting the first egress IP goes down, OCP will automatically switch to using the second (or third, etc) egress IP until the first egress IP's node comes back".

**Comment by jboxman** [comment]
@danwinship, I split up the content into separate sections with the additional information you provided.

What do you think?

Thanks.

**Comment by jboxman** [comment]
@danwinship - Does the latest version accurately reflect how egress IPs work?

Thanks!

**Comment by danwinship** [comment]
sorry for the delay. lgtm

File: modules/nw-egress-ips-about.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * networking/assigning-egress-ips.adoc
+
+[id="nw-egress-ips-about_{context}"]
+= Egress IP address assignment for project egress traffic
+
+By configuring an egressIP for a project, all outgoing external connections from the specified project will share the same, fixed source IP address.
```
Comment (ahardin-rh) [inline modules/nw-egress-ips-about.adoc]: s/egressIP/egress IP (Alternatively, if this is a variable, you can set it in backticks)

File: modules/nw-egress-ips-about.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Allowing additional IP addresses on the primary network interface might require extra configuration when using some cloud or VM solutions.
+====
+
+You can assign egress IP addresses to namespaces by setting the `egressIPs` parameter of the `NetNamespace` resource. After an egress IP is associated with a project, OpenShift SDN allows you to assign egress IPs to hosts in two ways:
+
+* In the _automatically assigned_ approach, an egress IP address range is assigned to a node.
+* In the _manually assigned_ approach, a list of one or more egress IP address is assigned to a node.
+
+Namespaces that request an egress IP address are matched with nodes that can host those egress IP addresses, and then the egress IP addresses are
+assigned to those nodes.
```
Comment (ahardin-rh) [inline modules/nw-egress-ips-about.adoc]: looks like a bad line break here

**Comment by ahardin-rh** [comment]
Just some minor comments from me. Otherwise, looks great!

File: modules/nw-egress-ips-about.adoc (hunk 1 above)
Comment (jboxman) [reply modules/nw-egress-ips-about.adoc]: Oops, supposed to just be _egress IP address_.

File: modules/nw-egress-ips-about.adoc (hunk 2 above)
Comment (jboxman) [reply modules/nw-egress-ips-about.adoc]: Fixed!

**Comment by jboxman** [comment]
@danwinship, no worries, thanks for your help with this!

**Comment by jboxman** [comment]
Hi @zhaozhanqi,

Can you confirm that the updates to the egress IP content in this PR are accurate?

Thanks!

**Comment by zhaozhanqi** [comment]
@huiran0826 please help check this. thx

**Comment by jboxman** [comment]
@huiran0826, thanks!

**Pull request merged**